spec/credit_card_bins_spec.rb

```diff
@@ -217,6 +217,13 @@
       expect(invalid_solo_bin.solo?).to eq(false)
     end
 
+    it "include a diners club? query" do
```

Comment:
If you break that scenario in two parts, could be better. 

what do you think ?